kernel/classes/ezcontentobjecttreenode.php

```diff
@@ -2906,6 +2906,11 @@ static function updateMainNodeID( $mainNodeID, $objectID, $version = false, $par
         }
 
         $db->commit();
+        eZAudit::writeAudit( 'main-node-update', array( 'Parent Node ID' => $parentMainNodeID,
+                                              'Main Node ID' => $mainNodeID,
```

Comment:
I think it might be possible to easily retrieve the old parent node id but not the old main node id without possibly causing cache issues and doing unnecessary extra operations.